_styles/style.css

```diff
@@ -1,3 +1,13 @@
+.Mealprep {
```

Comment:
You may need to change this from previous comment in index.html

Reply:
To separate with line .Mealprep from "{"and apply to all?

Reply:
If you changed the classname "Mealprep" to "meal-prep" then you would need to do in css file as well